app/main.py

```diff
@@ -2,0 +1,20 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks += 1
+    if masks:
+        return f"Friends should buy {masks} masks"
```

Comment:
The error message `Friends should buy {masks} masks` is not descriptive enough. According to the checklist, you should use descriptive error messages. It would be better to indicate which friends are not wearing masks. You could keep a list of friends who are not wearing masks and include their names in the error message. For example: `"{', '.join(friends_without_masks)} should buy masks"`.